index.html

```diff
@@ -1,232 +1,395 @@
+            <h3>Multi-Post Stories</h3>
+            <ul class="project-details-four">
+              <li>Canopy</li>
+              <li>BackEnd Dev</li>
+              <li>2015</li>
+            </ul>
+            <div>
+              <p>
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+            </div>
+            <div>
+              <ul class="languages-works-four">
+                <li>html</li>
+                <li>css</li>
+                <li>javascript</li>
+              </ul>
+            </div>
+            <button type="button" class="button">
+              <span>See Project</span>
+            </button>
+          </div>
+        </div>
+      </section>
+      <section class="about-me-skills">
+        <div class="about-me">
+          <div>
+            <h2>About Myself</h2>
+            <p>
+              I'm a software developer! I can help you build a product , feature
+              or website <br />
+              Look through some of my work and experience. <br />
+              If you like what you see and have a project you need coded, don’t
+              hestiate to contact me!
+            </p>
+          </div>
+          <div class="lets-connect-two">LET'S CONNECT</div>
+          <ul class="social-media-links">
+            <li>
+              <a
+                href="https://twitter.com/SharonVictor16_"
+                target="_blank"
+                rel="noopener"
+                class="social-media-link"
+                ><img src="./images/twitter-icon.svg" alt="Twitter" />
+              </a>
+            </li>
+            <li>
+              <a
+                href="https://www.linkedin.com/in/sharon-odhiambo-4333a0163/"
+                target="_blank"
+                rel="noopener"
+                ><img src="./images/linkedi-icon.svg" alt="LinkedIn" />
+              </a>
+            </li>
+            <li>
+              <a
+                href="https://www.linkedin.com/in/sharon-odhiambo-4333a0163/"
+                target="_blank"
+                rel="noopener"
+                ><img src="./images/github.svg" alt="LinkedIn" />
+              </a>
+            </li>
+            <li>
+              <a
+                href="https://www.linkedin.com/in/sharon-odhiambo-4333a0163/"
+                target="_blank"
+                rel="noopener"
+                ><img src="./images/medium.svg" alt="LinkedIn" />
+              </a>
+            </li>
+            <li>
+              <a
+                href="https://github.com/sharon-odhiambo"
+                target="_blank"
+                rel="noopener"
+                class="social-media-link"
+                ><img src="./images/angelist.svg" alt="Github" />
+              </a>
+            </li>
+          </ul>
+          <div>
+            <button type="button" class="button">
+              <span>Get my Resume</span>
```

Comment:
**[REQUIRED CHANGES]**

- Please, kindly improve your styling on all the `get my resume` buttons to match the Figma design below:



**figma**
![gmr](https://user-images.githubusercontent.com/59397159/172723241-94918fdf-be58-437c-8819-8cb319e941d0.png)





**your button**
![bone](https://user-images.githubusercontent.com/59397159/172722717-4916b239-ccb2-4fc5-a795-8245e272ce1a.png)